tests/near2far.cpp

```diff
@@ -137,7 +137,7 @@ int check_2d_3d(ndim dim, const double xmax, double a, component c0, component c
   continuous_src_time src(w);
   f.add_point_source(c0, src, zero_vec(dim));
   if (c1 != NO_COMPONENT) f.add_point_source(c1, src, zero_vec(dim), 0.7654321);
-  f.solve_cw(1e-6);
+  f.solve_cw(sizeof(realnum) == sizeof(float) ? 1e-5 : 1e-6);
```

Comment:
Probably not necessary to change this, but it's fine as long as the test still passes.